core/src/main/resources/error/error-classes.json

```diff
@@ -256,6 +256,11 @@
     ],
     "sqlState" : "22008"
   },
+  "DECIMAL_PRECISION_EXCEEDS_MAX_PRECISION" : {
```

Comment:
Does the error class have at least one test? If not, please add one. The same question about other new error classes.

Reply:
I am pretty sure there is one already: _LEGACY_ERROR_TEMP_2118.

Please double check before adding more new error messages.

Reply:
@amaliujia apart from INTEGER_OVERFLOW rest of the changes are merely replacing name for the existing class metaname
@MaxGekk all of these are already unit tested within relevant use cases via **intercept[]** but I can work on changing those cases to assert additionally the class name/msg using  **checkError( ... errorClass="CLASS_NAME")**

Reply:
> I can work on changing those cases to assert additionally the class name/msg using checkError ...

Please, do that. The purpose is to make the tests independent from error messages (only valuable message parameters), so, in this way tech editors could edit error message in error-classes.json and don't worry about internal Spark tests.

Reply:
sure, working on it

Reply:
@MaxGekk Can you please check if all good or not?